src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+}
+
+.header__container {
+  margin: 0 auto;
```

Comment:
You've used 'margin: 0 auto' to center the '.header__container'. It's not a mistake, but it's better to avoid fixing container sizes (unless there is a specific requirement). Let the content size dictate it to avoid overflow or an accidental scrollbar.